README.md

```diff
@@ -1,5 +1,81 @@
-# kfctl
+# Open Data Hub Operator
+
+## Introduction
+
+Open Data Hub operator is a downstream project of [kfctl](https://github.com/kubeflow/kfctl) operator that manages the 
+[KfDef](config/crd/bases/kfdef.apps.kubeflow.org_kfdefs.yaml) Custom Resource . This CR is used to deploy [Open Data Hub components](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/odh-core.yaml) in the
+OpenShift cluster. 
+
+## Usage
+
+### Installation
+
+The latest version of operator can be installed from the community-operators catalog on OperatorHub. It can also be build 
+and installed from source manually, see the Developer guide for further instructions.
+
+### Developer Guide
+
+**Pre-requisites**
+
+- Go version **go1.18.4** 
+- operator-sdk version can be updated to **v1.26.0**
```

Comment:
Testing out this operator version the kubebuilder version in `v1.26.0` requires go 1.19+.

We should just set this to operator-sdk 1.24.1